scripts/test-extensions-unit.sh

```diff
@@ -59,7 +59,8 @@ else
 				compile-extension:mssql \
 				compile-extension:notebook \
 				compile-extension:resource-deployment \
-				compile-extension:sql-database-projects
+				compile-extension:sql-database-projects \
```

Comment:
.bat version as well please 😄

Reply:
oh yeah, so used to mainly using mac 😅

Reply:
will add these in separate PR once we have tests as there is an issue with having no tests and adding this step in.